mycroft/skills/mycroft_skill/mycroft_skill.py

```diff
@@ -282,6 +272,23 @@ def settings_path(self):
             return self._old_settings_path
         return join(xdg.BaseDirectory.save_config_path(get_xdg_base(), 'skills', self.skill_id), 'settings.json')
 
+    @property
+    def settings(self):
+        if self._settings is not None:
+            return self._settings
```

Comment:
Consider returning a dict representation or else annotating this is a JsonStorage object

Reply:
JsonDatabase is a dict subclass, it is the same type just with extra methods for saving

The whole point is that it is a drop in replacement, it passes dict type checks and everything